pkg/catalogv2/system/system.go

```diff
@@ -227,22 +229,25 @@ func (m *Manager) Ensure(namespace, name, minVersion string, values map[string]i
 	return nil
 }
 
-func (m *Manager) Remove(namespace, name, minVersion string) {
-	delete(m.desiredCharts, desiredKey{
-		namespace:  namespace,
-		name:       name,
-		minVersion: minVersion,
-	})
+func (m *Manager) Remove(namespace, name string) {
```

Comment:
This will also nee unit tests.

Reply:
This is easy to test, yes.

Reply:
One thing to note here: this does effectively change the interface of this function. Before, you needed to know the previously request min version to remove from the desired charts. Now you just need to know the name/namespace. 

I think this is ok since the only function that was using remove seems to be the hosted stuff (eks/aks/gke), and it doesn't seem to specify a min version for install or remove (it is "" in both cases).

Reply:
This function is also not tested by integration tests.